reana_server/config.py

```diff
@@ -161,22 +162,23 @@ def _(x):
 # ===========================
 
 
-def _is_valid_rate_limit(rate_limit: str) -> bool:
-    return bool(
-        re.match(r"[0-9]+(\sper\s|\/)(second|minute|hour|day|month|year)", rate_limit)
-    )
```

Comment:
Never liked it ad happy to get rid of it, and replace it with `limits.util.parse`. Fewer things to worry about from our side.